src/prob/opf_bf.jl

```diff
@@ -0,0 +1,51 @@
+export run_opf_bf, run_ac_opf_bf, run_dc_opf_bf
+
+""
+function run_ac_opf_bf(file, solver; kwargs...)
+    return run_opf_bf(file, ACPPowerModel, solver; kwargs...)
+end
+
+""
+function run_dc_opf_bf(file, solver; kwargs...)
+    return run_opf_bf(file, DCPPowerModel, solver; kwargs...)
+end
+
+""
+function run_opf_bf(file, model_constructor, solver; kwargs...)
+    return run_generic_model(file, model_constructor, solver, post_opf_bf; kwargs...)
+end
+
+""
+function post_opf_bf(pm::GenericPowerModel)
+    variable_voltage(pm)
+    variable_generation(pm)
+    variable_branch_flow(pm)
```

Comment:
We should add branch current variables as well, right?

Reply:
current is also a flow variable, so I had put the initialization of the current variables at the same level as the calls to the active and reactive power initialization. will be addressed